java8/Java8.g4

```diff
@@ -348,7 +348,8 @@ fieldModifier
 	;
 
 variableDeclaratorList
-	:	variableDeclarator (',' variableDeclarator)*
+	:	variableDeclarator
+	|	variableDeclaratorList ',' variableDeclarator
```

Comment:
@KvanTTT How do fit no. of ATN states into the picture? I implemented a grammar transformation and had to make the decision whether I introduce new ATN states or new rules by alternatives.
As DFAs and NFAs are build up on demand during parsing, I was not aware of any other metric to take into account besides the ATN, which somehow indicates the language complexity.
So in my case also introducing new ATN states would have made more sense, right?